src/modules/samples/rust/src/sample.rs

```diff
@@ -19,17 +19,8 @@ const REPORTED_OBJECT_NAME: &str = "reportedObject";
 const DESIRED_ARRAY_OBJECT_NAME: &str = "desiredArrayObject";
 const REPORTED_ARRAY_OBJECT_NAME: &str = "reportedArrayObject";
 
-const STRING_SETTING_NAME: &str = "stringSetting";
-const INTEGER_SETTING_NAME: &str = "integerSetting";
-const BOOLEAN_SETTING_NAME: &str = "booleanSetting";
-const INTEGER_ENUMERATION_SETTING_NAME: &str = "integerEnumerationSetting";
-const STRING_ARRAY_SETTING_NAME: &str = "stringsArraySetting";
-const INTEGER_ARRAY_SETTING_NAME: &str = "integerArraySetting";
-const STRING_MAP_SETTING_NAME: &str = "stringMapSetting";
-const INTEGER_MAP_SETTING_NAME: &str = "integerMapSetting";
-
 // r# Denotes a Rust Raw String
-const INFO: &str = r#"""({
+const INFO: &str = r#""""({
```

Comment:
For this raw string, is it necessary to have the addtional `"""( ... )"""` at the beginning/end?  `r#" ... "` should be sufficient without the additional characters.